js/values/Paper.js

```diff
@@ -1,22 +1,14 @@
-var isUndefined = require( "lodash/lang/isUndefined" );
-
 /**
  * Construct the Paper object and set the keyword property.
- * @param {string} keyword The keyword to use in the analysis.
  * @param {string} text The text to use in the analysis.
- * @param {object} metaData The object containing all meta values.
+ * @param {object} attributes The object containing all attributes.
  * @constructor
  */
-var Paper = function( keyword, text, metaData ) {
-	this._keyword = keyword || "";
+var Paper = function( text, attributes ) {
 	this._text = text || "";
-
-	// if there is no metaData object given, use an empty object so checks on metaData keep working
-	if ( isUndefined( metaData ) ) {
-		metaData = {};
-	}
-
-	this._metaDescription = metaData.metaDescription || "";
+	this._attributes = attributes || {};
+	this._keyword = this._attributes.keyword || "";
+	this._metaDescription = this._attributes.metaDescription || "";
```

Comment:
This 'check' should probably be handled somewhere else in the code, seeing as it's not a hard requirement.